prov/tcp/src/tcpx.h

```diff
@@ -193,6 +200,10 @@ struct tcpx_ep {
 	fastlock_t		lock;
 	tcpx_ep_progress_func_t progress_func;
 	tcpx_get_rx_func_t	get_rx_entry[ofi_op_write + 1];
+	tcpx_hdr_ntoh_fn_t	hdr_ntoh_fn;
+	tcpx_hdr_hton_fn_t	hdr_hton_fn;
+	tcpx_hdr_sz_ntoh_fn_t   hdr_sz_ntoh_fn;
+	tcpx_hdr_flgs_ntoh_fn_t hdr_flgs_ntoh_fn;
```

Comment:
I think we can avoid this.   I don't see that flgs (is it too much to ask for the 'a' here?) is used.